source/_posts/routing-in-react.md

```diff
@@ -0,0 +1,590 @@
+---
+title: Routing in React
+contributor: Saurabh Ghatnekar
+avatar: saurabh-ghatnekar.png
+description: Learn how to use React Router to create a single-page application with multiple routes.
+date: 2023-02-17
+tags: [react, routing]
+id: routing-in-react
+relatedPost: [react, routing]
```

Comment:
This is not a place for tags, but for the related post.
- [ ] Use `react-security-best-practices`